coda/coda_replication/models.py

```diff
@@ -1,7 +1,7 @@
 from django.db import models
 # this is quite silly:
 # from django.conf import settings
-# import simplejson
+# import json
 # try:
 #     status_choice_json = open(settings.STATUS_CHOICE_FILE_PATH).read()
 #     status_choice_dict = simplejson.loads(status_choice_json)
```

Comment:
Need to replace simplejson with json here too

Reply:
yea...just in case if that code is brought back again..it will not have the unnecessary try..except

Reply:
I don't think that try/except block is like the others, I _think_ that's catching something else, but yeah, we do need to just do the simple replacement

Reply:
oh...yea...there's no try except here for the import. It'll error if it comes back.

Reply:
Is there a reason we don't just delete this block of commented out code?

Reply:
heh, I guess that should have been the first question here :smile:

Reply:
I think it can be deleted. (But didn't delete it just in case if it's supposed to be brought back later.)

Reply:
Let's delete it. I think it has been sitting there commented out for years.